src/globus_cli/commands/timer/create/transfer.py

```diff
@@ -161,30 +167,47 @@ def transfer_command(
 
     # Check endpoint activation, figure out scopes needed.
 
-    # check if either source or dest requires the data_access scope, and if so
-    # FIXME: hard fail for now (in the future, we should pick up on the requirement and
-    # generate a scope string to check against our logins)
-    source_epish = Endpointish(source_endpoint, transfer_client=transfer_client)
-    dest_epish = Endpointish(dest_endpoint, transfer_client=transfer_client)
-    needs_data_access: list[str] = []
-    if source_epish.requires_data_access_scope:
-        needs_data_access.append(str(source_endpoint))
-    if dest_epish.requires_data_access_scope:
-        needs_data_access.append(str(dest_endpoint))
-    if needs_data_access:
-        raise click.UsageError(
-            "Unsupported operation. 'globus timer create transfer' does not currently "
-            "support collections which use the data_access scope: "
-            f"{','.join(needs_data_access)}"
-        )
-
-    # Note this will provide help text on activating endpoints.
+    # the autoactivate helper may present output and exit in the case of v4 endpoints
+    # which need activation (e.g. OA4MP)
     autoactivate(transfer_client, source_endpoint, if_expires_in=86400)
     autoactivate(transfer_client, dest_endpoint, if_expires_in=86400)
 
-    # XXX: this section needs to be re-evaluated when 'timer create transfer' gets more
-    # capabilities to handle endpoints with scope requirements
-    # this would likely be the place to insert scope related checks
+    # if it is a client login or `--skip-data-access-check` has been used, skip the
+    # following handling for data_access
+    if not skip_data_access_check:
```

Comment:
The comment is out of date with the last point that I reached with this work. I believe it predates the `is_client_login()` check being included below.

My original thinking was that we didn't need to check `data_access` when client credentials are used, but if I recall correctly we do need to check and potentially request a new token.

Reply:
The `is_client_login()` check was present when I reviewed this. Based on the comment written directly above this, I'm wondering whether the condition should be something like:

```python
# if it is a client login or `--skip-data-access-check` has been used, skip
if not skip_data_access_check and not is_client_login():
    # suggested by comment:   ^^^^^^^^^^^^^^^^^^^^^^^^^
    ...
```

Reply:
I need to double-check this, but I think the current logic is more right than the comment. But I'll circle back on this and really sort it out before saying more, since I don't remember what's here perfectly.